crates/proof-of-sql/src/base/database/column.rs

```diff
@@ -206,15 +228,15 @@ impl ColumnType {
                 | ColumnType::BigInt
                 | ColumnType::Int128
                 | ColumnType::Scalar
-                | ColumnType::Decimal75(_, _)
+                | ColumnType::Decimal75(_, _) // TODO: is a timestamp numeric?
         )
     }
 
     /// Returns true if this column is an integer and false otherwise
     pub fn is_integer(&self) -> bool {
         matches!(
             self,
-            ColumnType::SmallInt | ColumnType::Int | ColumnType::BigInt | ColumnType::Int128
+            ColumnType::SmallInt | ColumnType::Int | ColumnType::BigInt | ColumnType::Int128 // TODO: is a timestamp an integer?
```

Comment:
No since I'm not sure whether it makes any sense to do integer operation on it. If we add Duration type though it can become useful since Timestamp + Duration and Timestamp - Duration have meanings.